conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+                <li>any evidence or applicable materials relating to the conflict (e.g., emails, minutes, screenshots, or other materials)</li>
+                <li>what resolution or action you wish to be taken to address the conflict</li>
+            </ul>
+
+            <p>An investigator will be appointed and will talk to both parties and anyone else relevant. The investigator's role is to:</p>
+
+            <ul>
+                <li>establish what happened, in a fair and objective manner</li>
+                <li>ensure that the investigation is proportionate to the seriousness and complexity of the complaint</li>
+                <li>ensure that the investigation is concluded without delay</li>
+                <li>keep notes and submitted evidence in confidence</li>
+            </ul>
+
+            <p>The investigator will write a summary of the result of the investigation and recommend what actions should be taken. Outcomes and actions will be agreed by W3M and communicated to both parties.</p>
+        </section>
+
+        <section id="disciplinary-actions">
+            <h2>Disciplinary Actions</h2>
+
+            <p>There are various actions that the investigator can recommend, depending on the frequency and severity of the behaviours in conflict with the CoC.</p>
+
+            <p>The intent of the conflict resolution procedures is to ensure that individuals can continue to participate in W3C activities.</p>
+
+            <p>Where it is beneficial to the continued functioning of W3C, a high level summary of results and actions from an investigation may be communicated to the membership, for example if an individual is suspended from a working group or from being a chair in any working group. Otherwise, results and actions should be kept confidential.</p>
+
+            <p>Chairs, ombudspeople, mediators, and investigators can recommend the following actions be taken if resolution is not possible:</p>
+
+            <ol>
+                <li>Corrective action. The individual who had the complaint raised against them will be advised to correct their behaviour going forward, for example not engaging in the same behaviours that resulted in the complaint.</li>
+                <li>Agree to a mediation process. The individual who had the complaint raised against them will be advised to participate in a mediation process, where both parties can agree to certain actions and changes going forward. </li>
+                <li>An official warning. In cases where the actions by an individual are frequent or severe, an individual and their employer (if relevant) will be given an official warning that their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, this may result in the removal or suspension of their W3C participation. There is no time limit for the expiry of the warning.</li>
```

Comment:
s/actions by an individual/actions by an offender/

s/an individual and their employer/the offender and their employer/

s/they continue/the behaviors/